deepchem/feat/molecule_featurizers/mol_graph_conv_featurizer.py

```diff
@@ -247,10 +247,13 @@ def _featurize(self, datapoint: RDKitMol, **kwargs) -> GraphData:
 
       for x, y, z in zip(pos_x, pos_y, pos_z):
         pos.append([x, y, z])
+      node_pos_features = np.asarray(pos)
+    else:
+      node_pos_features = None
     return GraphData(node_features=atom_features,
                      edge_index=np.asarray([src, dest], dtype=int),
                      edge_features=bond_features,
-                     pos=np.asarray(pos))
+                     node_pos_features=node_pos_features)
```

Comment:
Could you clarify how having pos and node_pos_features both equalling np.array(pos) resolves the redundant pos kwarg that is occuring in issue 3090? Thank you.

Reply:
The `pos` is a kwarg to `GraphData` and in these lines https://github.com/deepchem/deepchem/blob/3509f3c27518073849f34131cbc5f8b963f6128b/deepchem/feat/graph_data.py#L149-L154 we are passing `pos` as an kwarg as well as a positional argument to `Data`. It should ideally by `node_pos_features` since `node_pos_features` stores position of nodes.